.cz.json

```diff
@@ -12,7 +12,7 @@
       "message_template": "{{change_type}}:{% if show_message %} {{message}}{% endif %}",
       "example": "feat: this feature enable customize through config file",
       "schema": "<type>: <body>",
-      "schema_pattern": "^((build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\\([\\w\\-\\.]+\\))?(!)?: ([\\w \\-'])+([\\s\\S]*))|Merge.*",
+      "schema_pattern": "^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test){1}(\\([\\w\\-\\.]+\\))?(!)?: ([\\w \\-'])+([\\s\\S]*)",
```

Comment:
Is it abandoning `staging` that makes the `Merge` subpattern no longer necessary here? We're 100% rebase on updating out-of-date feature branches?

Reply:
Commitizen has an [`allowed_prefixes`](https://commitizen-tools.github.io/commitizen/config/#allowed_prefixes) option which defaults to allowing merges, so we don't need the schema to match the Github ruleset